pyvista/core/dataset.py

```diff
@@ -391,7 +392,7 @@ def points(self) -> pyvista_ndarray:
         return pyvista_ndarray(_points, dataset=self)
 
     @points.setter
-    def points(self, points: np.ndarray):
+    def points(self, points: Union[pyvista_ndarray, PointLike]):
```

Comment:
`pyvista_ndarray` is a subclass of np.ndarray which is included in `PointLike`, so do we need to explicitly mention this here? Alternatively we should add our subclass to the original definition of `PointLike`.

Reply:
I got myself turned around on inheritance and typing.  You are right the original typing was correct.